worldedit-core/src/main/java/com/fastasyncworldedit/core/util/MathMan.java

```diff
@@ -75,7 +75,7 @@ public static int clamp(int check, int min, int max) {
     }
 
     public static float clamp(float check, float min, float max) {
-        return check > max ? max : (Math.max(check, min));
+        return check > max ? max : Math.max(check, min);
```

Comment:
But like... why is this a thing in the first place?

Reply:
#clamp is used in TextureUtil, if you have concerns to address dated or unused methods in the math classes, I recommend to address them separately, because there are a lot.